README.md

```diff
@@ -14,7 +14,7 @@ of kiwix.
 Most of the compilation steps are handled by the kiwix-build.py
 script.
 
-This script has been tested of Fedora 23 and Ubuntu 16.10
+This script has been tested on Fedora 23, Ubuntu 16.10 and Debian 8
```

Comment:
In this case we should not say it is tested on Debian 8.
This sentence is a kind of implicit contract. It implies that the script is intended to work on Fedora 23 and Ubuntu 16.10 (and that we will fix potential bug)